lms/envs/devstack_docker.py

```diff
@@ -15,7 +15,7 @@
 
 ECOMMERCE_PUBLIC_URL_ROOT = 'http://localhost:18130'
 ECOMMERCE_API_URL = 'http://edx.devstack.ecommerce:18130/api/v2'
-
+ENTERPRISE_API_URL = 'http://enterprise.example.com/enterprise/api/v1/'
```

Comment:
Why is this here? We should *not* have fake settings in the Docker settings. If the Enterprise API is not enabled by default, this needs to go away.